### ISSUE TITLE
scene_delaunay: not working

### ISSUE THREAD

**Issue by wflieller** [djnavarro/jasmines#6]
Keep getting error when trying to use the delaunay scene.

`Error: 'runifpoint' is not an exported object from 'namespace:spatstat'`

Looked every where for a solution but can't get it to work.

**Comment by djnavarro** [comment]
Yep, the problem is at my end. There was a breaking change in spatstat a little while ago that affects `scene_delaunay()` and I haven't had a chance to fix it yet. I do intend to fix it, but for the moment this is broken
